index.js

```diff
@@ -408,6 +408,53 @@ require('yargs')
                 Logger.error(err.message);
             });
     })
+    .command('decorate', 'apply the decorators and vocabs to the target models from given list of dcs files and vocab files', yargs => {
+        yargs.demandOption('models', 'Please provide a model');
+        yargs.option('models', {
```

Comment:
Should this be singular, as per the docs above? Or otherwise we support both?

Reply:
I think we should support both. will update the PR accordingly.

Reply:
Updated PR.